src/Crate/DBAL/Driver/PDOCrate/PDOConnection.php

```diff
@@ -36,6 +37,7 @@ class PDOConnection extends PDO implements ServerInfoAwareConnection
     public function __construct($dsn, $user = null, $password = null, array $options = null)
     {
         parent::__construct($dsn, $user, $password, $options);
+        $this->setAttribute(PDO::ATTR_STATEMENT_CLASS, [Statement::class, []]);
```

Comment:
not sure if crate supports this. See also #112 and https://github.com/doctrine/dbal/issues/4619#issuecomment-827900550

Reply:
[1] shows lots of `Crate\PDO\Exception\PDOException: Unsupported driver attribute` errors. Can you make sense of that and evaluate whether corresponding support can be added to `crate-pdo`?

[1] https://github.com/crate/crate-dbal/runs/2452374425#step:9:17

Reply:
Not 100% sure but I think something like this is required: https://github.com/crate/crate-pdo/pull/121

Reply:
Thank you for https://github.com/crate/crate-pdo/pull/121. It has just converged into [`crate-pdo==2.1.0`](https://packagist.org/packages/crate/crate-pdo#2.1.0).

Reply:
Thank you did remove the `composer.lock` here also. As we will now need to require `2.1` I think we will run into problems with `7.2`  as the crate-pdo 2.1.0 does not support `7.2` how should we handle this?

Reply:
Isn't PHP 7.2 end of life? We already dropped support on this with crate-pdo 2.1.0 on purpose. So, let's also carry this forward here?

Reply:
yes 7.2 is end of life. I'm fine with dropping it here also https://www.php.net/supported-versions.php